server/routes/viewModels/taskLists/tasks/riskManagement.js

```diff
@@ -1,19 +1,34 @@
 const { standardAction, viewEdit, view } = require('./utils/actions')
 
 const getLabel = ({ decisions, tasks }) => {
-  const { addressUnsuitable, awaitingRiskInformation, riskManagementNeeded } = decisions
+  const {
+    riskManagementVersion,
+    riskManagementNeededV1,
+    mandatoryAddressChecksNotCompletedV2,
+    addressUnsuitable,
+    awaitingRiskInformation,
+    bassAreaSuitable,
+  } = decisions
   const { riskManagement } = tasks
 
   if (addressUnsuitable) {
     return 'Address unsuitable'
   }
 
+  if (mandatoryAddressChecksNotCompletedV2 && !bassAreaSuitable) {
+    return 'WARNING||Mandatory address checks not completed'
+  }
+
   if (awaitingRiskInformation) {
     return 'WARNING||Still waiting for information'
```

Comment:
is this ever shown for v2? 

maybe it would be clearer to split the logic by version at the top level? (
even if there is a small amount of non complex duplication)
```
if (v1) {
   if (a) return blah
   else return  bleh
}
if (v2) {
   if (a) return bluh
   else return  bloh
}
```

Otherwise all the way through, have to have version concerns playing around in the back of my mind and also whether these fields can even exist for a given version, whether there are missed cases etc..

Reply:
(suggesting addressing maybe as a follow up)